vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: vCluster Pro air-gapped install
+sidebar_label: Air-gapped
+sidebar_position: 5
+description: Learn how to install vCluster Pro in an air-gapped Kubernetes cluster without the vCluster Platform agent.
+sidebar_class_name: pro
+---
+
+import BasePrerequisites from '../../../platform/_partials/install/base-prerequisites.mdx';
+
+import FipsConfig from '../../_partials/deploy/fips-config.mdx';
+
+# vCluster Pro air-gapped install
+
+This document details the prerequisites and steps to install vCluster Pro into
+an `air-gapped` Kubernetes cluster without the platform agent running on the host
```

Comment:
I don't think we need to mention without the platform agent running on the host. This is the vCluster standalone docs, so I would assume that these docs don't have any platform agent running on it.